extension-manifest-v3/src/utils/engines.js

```diff
@@ -221,7 +221,7 @@ function notifyListeners(name) {
   });
 }
 
-const CDN_HOSTNAME = chrome.runtime.getManifest().debug
+const CDN_HOSTNAME = stagingMode
```

Comment:
in the metrics we use debugMode instead - I think we should do it here as well.

Reply:
Sure. I think I did that looking for URLs to the CDN. 

As we discuss that, I am considering if we should add then the debug mode flag into the main `npm start` script instead of another command. When the staging engines was combined with debug mode, it wasn't a good idea. 

Now, I think, that we could do that for access to those hard to test features by default, and avoid sending pings/WTM signals to the production environment when developing.

Reply:
reading this and other comments convinces me that we should have `stagingMode` here and in other places the configure endpoints